glusterd2/commands/peers/peer-rpc-svc.go

```diff
@@ -26,20 +27,25 @@ func (p *PeerService) RegisterService(s *grpc.Server) {
 
 // Join makes the peer join the cluster of the requester
 func (p *PeerService) Join(ctx context.Context, req *JoinReq) (*JoinRsp, error) {
-	logger := log.WithField("remotepeer", req.PeerID)
+	logger := log.WithFields(log.Fields{
+		"remotepeer":    req.PeerID,
+		"remotecluster": req.ClusterID})
 
 	logger.Info("handling new incoming join cluster request")
 
 	// Handling a Join request happens as follows,
 	// 	- TODO: Ensure no ongoing operations (transactions/other peer requests) are happening
-	// 	- TODO: Check is peer is part of another cluster
```

Comment:
This is actually a step in the the list of steps to check, and should remain for the whole comment to make sense. Just remove the 'TODO:' prefix.

Reply:
Oops! `Check is` should have been `Check if`. I will sneak in the grammatical fix in one of the future PRs.